WalletWasabi.Gui/Shell/Commands/ToolCommands.cs

```diff
@@ -45,10 +65,43 @@ private void OnWalletManager()
 			}
 		}
 
+		private void OnEncryptionManager()
+		{
+			var encryptionManagerViewModel = IoC.Get<IShell>().GetOrCreate<EncryptionManagerViewModel>();
+			encryptionManagerViewModel.SelectTab(EncryptionManagerViewModel.Tabs.Sign);
+		}
+
 		[ExportCommandDefinition("Tools.WalletManager")]
 		public CommandDefinition WalletManagerCommand { get; }
 
+		[ExportCommandDefinition("Tools.EncryptionManager")]
+		public CommandDefinition EncryptionManagerCommand { get; }
+
 		[ExportCommandDefinition("Tools.Settings")]
 		public CommandDefinition SettingsCommand { get; }
+
+		#region IDisposable Support
+
+		private volatile bool _disposedValue = false; // To detect redundant calls
+
+		protected virtual void Dispose(bool disposing)
+		{
+			if (!_disposedValue)
+			{
+				if (disposing)
+				{
+					Disposables.Dispose();
```

Comment:
The recommendation used to be to set the disposed object references to null. In this case:
```c#
Disposable = null;
```
I used to think that was silly but it prevents us to use disposed object by mistake (because then we get a NRE). I would do it here.

Reply:
This is a good idea. Last time I discovered a bug because I set the large fields to null. However in this case Code Cracker suggestion is to do 
`private CompositeDisposable Disposables { get; } = new CompositeDisposable();`
In this case the property is readonly and cannot set to null. 
Because we are following Code Cracker directives this won't be nulled out.